### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by dev-manoranjan**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,4 +6,4 @@ import {AppRegistry} from 'react-native';
 import App from './App';
 import {name as appName} from './app.json';
 
-AppRegistry.registerComponent(appName, () => App);
```
Comment (kmgaurav2611) [inline index.js:9]: I have renamed the app in the app.json file. So we don't need this here.

File: src/screens/saved-jobs/index.tsx [hunk 2]
```diff
@@ -41,8 +41,14 @@ const SavedJobs = ({navigation}) => {
   };
 
   const getData = async () => {
-    const resp = await callService(ApiMethods.GET, ENDPOINT.GET_MENTORS);
+    const resp = await callService(
+      ApiMethods.GET,
+      `${ENDPOINT.SAVED_JOBS}test.user@gmail.com`,
```
Comment (kmgaurav2611) [inline src/screens/saved-jobs/index.tsx:46]: Do we need to append an email to the endpoint ?

File: index.js (hunk 1 above)
Comment (dev-manoranjan) [reply index.js:9]: without renaming I can't run the project

File: src/screens/saved-jobs/index.tsx (hunk 2 above)
Comment (dev-manoranjan) [reply src/screens/saved-jobs/index.tsx:46]: Yes, I think and it's not working yet, so pass it later will check

**Review by Rajesh4191 — APPROVED** [review]

**Pull request merged**